cypress/e2e/base.spec.ts

```diff
@@ -20,4 +20,94 @@ describe('Visit directory page and verify users exists', () => {
   });
 });
 
+describe('directory infinite scroll', () => {
+  const PAGE_LIMIT = 20;
```

Comment:
I know we use this here `server\api\users\users.controller.ts` too so should probably be abstracted away to an env property or something?

Is env the best way to handle app settings for something like `page_limit`?